script.js

```diff
@@ -0,0 +1,82 @@
+let Library = null;
+function displayBooks() {
+  const section = document.getElementById('bookList');
+  const container = document.createElement('ul');
+  container.id = 'container';
+  Library.forEach((content) => {
+    const { title } = content;
+    const { id } = content;
```

Comment:
- [ ] OPTIONAL 
When 2 or more books with **same** _book name_ and _book author_  are stored and then a single entry having same details is removed all other entry with the same details also gets removed. You can prevent this by ensuring that each entry has a unique `id` or check if a  duplicate exists and prevent entry